src/basecamp.cpp

```diff
@@ -100,10 +100,16 @@ void basecamp::add_expansion( const std::string &terrain, const tripoint &new_po
 
 void basecamp::define_camp( npc &p )
```

Comment:
We need to keep this old version, because there might be some people who have updated their game in forever and have a savegame from before I starting merging basecamps and faction camps.